kafl_fuzzer/common/config/settings.py

```diff
@@ -28,9 +28,14 @@ def app_settings_files() -> List[str]:
     ]
     # env var KAFL_CONFIG_FILE if present
     try:
-        settings_files.append(os.environ['KAFL_CONFIG_FILE'])
+        env_config_file = os.environ['KAFL_CONFIG_FILE']
```

Comment:
How about `os.environ.get('KAFL_CONFIG_FILE', None)` to avoid the try/except/else?

Reply:
Agreed.
Although `try/except` is more Pythonic, in this case we don't do anything in the except statement.